onepassword/resource_onepassword_item.go

```diff
@@ -9,16 +9,60 @@ import (
 	"github.com/1Password/connect-sdk-go/onepassword"
 	"github.com/hashicorp/go-uuid"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 )
 
+const (
+	itemUUIDDescription  = "The UUID of the item. Item identifiers are unique within a specific Vault."
+	vaultUUIDDescription = "The UUID of the vault the item is in."
+	categoryDescription  = "The category of the item."
+	itemTitleDescription = "The title of the item."
+	urlDescription       = "The primary URL where the resource is applicable."
+	tagsDescription      = "An array of strings of the tags assigned to the item."
+	usernameDescription  = "Username for this item."
+	passwordDescription  = "Password for this item."
+
+	dbHostnameDescription = "(Only applies to the database category) Where can the database be found."
+	dbDatabaseDescription = "(Only applies to the database category) The name of the database."
+	dbPortDescription     = "(Only applies to the database category) The port is the database listening on."
+	dbTypeDescription     = "(Only applies to the database category) The type of database."
+
+	sectionsDescription      = "A list of custom sections on an item"
+	sectionDescription       = "A section on an item that contains custom fields"
+	sectionIDDescription     = "A unique identifier for the section."
+	sectionLabelDescription  = "The label for the section."
+	sectionFieldsDescription = "A list of custom fields in this section."
+
+	fieldDescription        = "A custom field."
+	fieldIDDescription      = "A unique identifier for the field."
+	fieldLabelDescription   = "The label for the field."
+	fieldPurposeDescription = "Purpose indicates this is the special username, password, or notes field"
```

Comment:
This is tricky, but does this read well to you? There might be more I can do.

```suggestion
	fieldPurposeDescription = "Purpose indicates this is a special field: a username, password, or notes field."
```